README.md

```diff
@@ -1,5 +1,9 @@
 # Tachyons Bootstrap
 
+[![Build Status](https://travis-ci.org/dwyl/tachyons-bootstrap.svg?branch=master)](https://travis-ci.org/dwyl/tachyons-bootstrap)
+
+[![codecov](https://codecov.io/gh/dwyl/tachyons-bootstrap/branch/master/graph/badge.svg)](https://codecov.io/gh/dwyl/tachyons-bootstrap)
```

Comment:
if you remove the linebreak here the badges will appear on the same line in the _rendered_ page.